ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/builder/ElectraBuilder.java

```diff
@@ -180,6 +188,30 @@ public ElectraBuilder maxPendingDepositsPerEpoch(final Integer maxPendingDeposit
     return this;
   }
 
+  public ElectraBuilder maxBlobsPerBlockElectra(final Integer maxBlobsPerBlockElectra) {
+    checkNotNull(maxBlobsPerBlockElectra);
+    this.maxBlobsPerBlockElectra = maxBlobsPerBlockElectra;
+    return this;
+  }
+
+  public ElectraBuilder targetBlobsPerBlockElectra(final Integer targetBlobsPerBlockElectra) {
+    checkNotNull(targetBlobsPerBlockElectra);
+    this.targetBlobsPerBlockElectra = targetBlobsPerBlockElectra;
+    return this;
+  }
+
+  public ElectraBuilder blobSidecarSubnetCountElectra(final Integer blobSidecarSubnetCountElectra) {
+    checkNotNull(blobSidecarSubnetCountElectra);
+    this.blobSidecarSubnetCountElectra = blobSidecarSubnetCountElectra;
+    return this;
+  }
+
+  public ElectraBuilder maxRequestBlobSidecarsElectra(final Integer maxRequestBlobSidecarsElectra) {
+    checkNotNull(maxRequestBlobSidecarsElectra);
+    this.maxRequestBlobSidecarsElectra = maxRequestBlobSidecarsElectra;
+    return this;
+  }
```

Comment:
Nit: to match the param order, `maxRequestBlobSidecarsElectra` should be above `blobSidecarSubnetCountElectra`.

Reply:
Done